tests/data/species_trees/primates.tre

```diff
@@ -0,0 +1 @@
+(((human:5.6,chimpanzee:5.6):3.0,gorilla:8.6):9.4,orangutan:18.0)
```

Comment:
Not really worth storing these as files; we could just keep them as module level variables in ``test_species_tree.py``, like
```python
primates_tree = "(((human:5.6,chimpanzee:5.6):3.0,gorilla:8.6):9.4,orangutan:18.0)"
```

Reply:
Done, except for two large input trees (one in starbeast format, one with over 100 species).